### DISCUSSION TITLE
Feature request: Add labels directly from title or comment

### DISCUSSION THREAD

**Discussion by radulucut** [community/community#21715] [New to GitHub]
<p>Adding labels can be (feel) a bit slow when you quickly want to create multiple issues.</p>
<p>Having the abillty to declare them directly into the title or comment and work as you’ve added them from the labels section could be a handy feature.</p>
<p>This could be applied for assignees, projects or other stuff.</p>
<p>Something like:</p>
<p><img src="https://user-images.githubusercontent.com/13837994/181066429-ec17c9e0-5ef7-47ad-bd6d-81a9f75f4239.png" alt="labels.PNG" data-base62-sha1="tUx5O5GWUZo8t9XQ06BjjAHcpBi" width="663" height="402"></p>
<p>What do you think?</p>

**Comment by MChevy422** [comment, marked as answer]
<p><strong>Hello <span><a class="mention" href="/radulucut">@radulucut</a></span> <br><br>Thank you for your feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.<br><br>Once again, thank you for your input!<br><br>Greatly appreciated<br><br>MChevy422</strong></p>
